src/token/src/canister.rs

```diff
@@ -343,39 +407,6 @@ impl TokenCanister {
         self.state.borrow_mut().bidding_state.auction_period = period_sec * 1_000_000;
         Ok(())
     }
-
-    /*********************** NOTIFY **********************/
-
-    /// Notifies the transaction receiver about a previously performed transaction.
-    ///
-    /// This method guarantees that a notification for the same transaction id can be sent only once.
-    /// It allows to use this method to reliably inform the transaction receiver without danger of
-    /// duplicate transaction attack.
-    ///
-    /// In case the notification call fails, an [TxError::NotificationFailed] error is returned and
-    /// the transaction will still be marked as not notified.
-    ///
-    /// If a notification request is made for a transaction that was already notified, a
-    /// [TxError::AlreadyNotified] error is returned.
-    #[update]
-    async fn notify(&self, transaction_id: Nat) -> TxReceipt {
-        notify(self, transaction_id).await
-    }
-
-    /// Convenience method to make a transaction and notify the receiver with just one call.
-    ///
-    /// If the notification fails for any reason, the transaction is still completed, but it will be
-    /// marked as not notified, so a [notify] call can be done later to re-request the notification of
-    /// this transaction.
-    #[update]
-    async fn transferAndNotify(
-        &self,
-        to: Principal,
-        amount: Nat,
-        fee_limit: Option<Nat>,
-    ) -> TxReceipt {
-        transfer_and_notify(self, to, amount, fee_limit).await
-    }
```

Comment:
Since Dfinity found the way to fix and not depricate the notify method, we want to keep it also.